src/vma/main.cpp

```diff
@@ -269,16 +269,20 @@ void check_debug()
 void check_flow_steering_log_num_mgm_entry_size()
 {
 	char flow_steering_val[2] = {0};
-	if (priv_read_file((const char*)FLOW_STEERING_MGM_ENTRY_SIZE_PARAM_FILE, flow_steering_val, 2) == -1) {
+	if (priv_read_file((const char*)FLOW_STEERING_MGM_ENTRY_SIZE_PARAM_FILE, flow_steering_val, 2, VLOG_DEBUG) == -1) {
 		vlog_printf(VLOG_DEBUG, "Flow steering option does not exist in current OFED version");
```

Comment:
improve error message by adding 'mlx4'
"Flow steering option **for mlx4 driver** does not exist in current OFED version"